src/applications/gi/containers/GiBillApp.jsx

```diff
@@ -85,7 +86,10 @@ export class GiBillApp extends React.Component {
     if (constants.inProgress) {
       content = <LoadingIndicator message="Loading..." />;
     } else {
-      content = this.props.children;
+      content =
+        constants.statusCode === (503 || 500 || 504)
```

Comment:
Need descriptive constants for the error codes here and in other files, so we don't have magic numbers.